NotFound.vue

```diff
@@ -16,23 +16,17 @@ const msgs = [
   `Looks like we've got some broken links.`
 ];
 
+import RedirectMixin from './mixins/RedirectMixin';
```

Comment:
Why we can't use the component in the template?

Reply:
I used mixin because the same logic is used in NotFound component. It is used to redirect links like `/converting_existing_servers.html`

Reply:
What if I add
<Redirect/>
in the NotFound template? Is it not working?

Reply:
It works. But you will see Not Found page content for a moment before redirecting happens.